app/controllers/events_controller.rb

```diff
@@ -7,7 +7,7 @@ class EventsController < ApplicationController
 
   # GET /events
   def index
-    @events = Event.sorted_by_start_date(!can?(:edit, Event)).reverse
+    @events = Event.sorted_by_start_date(!can?(:view_unpublished, Event))
```

Comment:
I dont think it is good to have the latest (and therefore probably most uninteresting) events shown first and events that are about to begin (and therefore interesting) towards the bottom. Also, this achieves consistency with the start page. 
Is there a reason for showing them in different orders on start page and event page @tom95 @hpi-swt2/product-owner-2016 ?